packages/flame_splash_screen/README.md

```diff
@@ -0,0 +1,145 @@
+# Flame Splash Screen
+
+<!-- markdownlint-disable MD013 -->
+<p align="center">
+  <a href="https://flame-engine.org">
+    <img alt="flame" width="200px" src="https://user-images.githubusercontent.com/6718144/101553774-3bc7b000-39ad-11eb-8a6a-de2daa31bd64.png">
+  </a>
+</p>
+
+Style your [Flame](https://github.com/flame-engine/flame) game with a beautiful splash screen.
+
+This package includes a `FlameSplashScreen` widget.
+
+<p align="center">
+  <a title="Pub" href="https://pub.dev/packages/flame_splash_screen" ><img src="https://img.shields.io/pub/v/flame_splash_screen.svg?style=popout" /></a>
+  <a title="Test" href="https://github.com/flame-engine/flame/actions?query=workflow%3Acicd+branch%3Amain"><img src="https://github.com/flame-engine/flame/workflows/cicd/badge.svg?branch=main&event=push"/></a>
+  <a title="Discord" href="https://discord.gg/pxrBmy4"><img src="https://img.shields.io/discord/509714518008528896.svg"/></a>
+  <a title="Melos" href="https://github.com/invertase/melos"><img src="https://img.shields.io/badge/maintained%20with-melos-f700ff.svg"/></a>
+</p>
+
+<p align="center">
+  <img src="demo.gif" />
+</p>
+
+
+## Install
+
+Add `flame_splash_screen` as a dependency in your pubspec.yaml file ([what?](https://flutter.io/using-packages/)).
```

Comment:
nit: https://docs.flutter.dev/packages-and-plugins/using-packages
I would also just link that over `your pubspec.yaml file`